src/content/en/2020/seo.md

```diff
@@ -23,3 +23,835 @@ queries: 07_SEO
+
+The most frequently used directives focused on search engines and their paid marketing counterparts. SEO tools Ahref and Majestic were in the top five disallow statements for both devices.
+
+<figure markdown>
+User-agent | % of mobile robots.txt | % of desktop robots.txt
+-- | -- | --
+`*` | 74.40% | 73.16%
+adsbot-google | 5.63% | 5.68%
+mediapartners-google | 5.55% | 3.83%
+mj12bot | 5.49% | 5.30%
+ahrefsbot | 4.80% | 4.66%
+
+<figcaption>{{ figure_link(caption="Robots.txt user-agent directives.", sheets_gid="TODO", sql_file="TODO.sql") }}</figcaption>
+</figure>
+
+
+
+When analyzing the usage of the disallow statement in robots.txt by using Lighthouse-powered data of over 6 million sites, it was found that 97.84% of them were completely crawlable, with only 1.05% using a disallow statement.
+
+An analysis of the robots.txt disallow statement usage along the meta robots indexability directives was also done, finding 1.02% of the sites including a disallow statement along indexable pages -featuring a meta robots index tag-, with only 0.03% of sites using the disallow statement in robots.txt along noindexed page via the meta robots noindex directive.
+
+This is notable as [Google documentation](https://developers.google.com/search/docs/advanced/robots/intro) states sites should not use robots.txt as a means to hide web pages from Google Search as internal linking with descriptive text could result in the page being indexed without a crawler visiting the page, and use instead other methods, like a `noindex` directive via meta robots.
+
+#### Meta Robots
+
+Robots meta tag and X-Robots-Tag in HTTP headers are an extension of the proposed [Robots Exclusion Protocol](https://webmasters.googleblog.com/2019/07/rep-id.html) (REP) which allow directives to be allowed at a more granular level. Directive support varies by search engine as REP is not yet an official internet standard.
+
+Meta tags were the dominant method of granular execution with 27.70% of desktop and 27.96% of mobile pages using the tag. X-Robots-Tag directives were found on 0.27% and 0.40% of desktop and mobile respectively.
+
+{{ figure_markup(
+  image="seo-on-page-robots-directive-use.png",
+  caption="On-Page Robots Directive Use",
+  description="Bar chart showing robots usage. Meta-Robots is 27.70% for desktop and 27.96% for mobile, X-Robots barely registers on the chart with a mere 0.27% of pages on desktop and 0.40% on mobile",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=99993402&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sqll"
+  )
+}}
+
```

Comment:
I raised a question as to whether the use of an HTTP Header should really be considered as "On-Page"? Yes it's in control of the site, but not really on the page. I think "Robots Directive Usage" is sufficient thus avoiding this problem.

WDYT?

Reply:
Hi @bazzadp, thanks for raising this! I think is because of the "On-Page" meaning in the "SEO context": "On-Page" is whatever happens on your site and "Off-Page" whatever happens outside your site (external incoming links), so within this context the "HTTP Headers" would be "On-Page" rather than "Off-Page". However, to avoid misunderstandings, it's ok to remove the "On-Page" term from the image caption here, I don't think it will take anything from it and leaving just "Robots Directive Use" should be ok. If you think it's necessary, just let me know and I'll remove the "On-Page" word. Thanks again :)

Reply:
Yes I thought that might be what you were meaning with "on-page". Personallyyes I think it's a little confusing and, as this is the only graph you've used that phrase for (and the only one where it's arguably not 100% accurate), I'd remove it just to avoid any confusion.

@antoineeripret would need to update the chart and also export the image again. But since he;'s going to do that to add labels he can do this at same time.

Reply:
Perfect! Thanks @bazzadp @antoineeripret - let's remove the "on-page" word then :) Thanks again

Reply:
I got edit access to the Sheet so this is fixed.

Reply:
@bazzadp : I'm just seeing this right now, you already generated the new graph, right? No no further action needed from my side on that particular discussion?

Reply:
Correct all done. Didn't have access previously so needed you to do it, but managed to get access and so took care of it. Similar with regeneration of some of the images - that's all sorted now too.

Reply:
Awesome! Thanks a lot!